odata-csdl/1 Introduction.md

```diff
@@ -494,6 +494,305 @@ representation of primitive type values in URLs and
 [OData-JSON](#ODataJSON) for the representation in requests and
 responses.
 
+## ##subsec Type Facets
+
+Facets modify or constrain the acceptable values for a model element of a given primitive type,
```

Comment:
I initially read this as the model element belonging to a given primitive type, not that the model element was a given primitive type.

Perhaps turn this around; something like:
```suggestion
Facets modify or constrain the acceptable values of primitive typed model elements,
```